hudi-integ-test/src/main/java/org/apache/hudi/integ/testsuite/generator/GenericRecordFullPayloadGenerator.java

```diff
@@ -284,7 +284,7 @@ private Object typeConvert(Schema.Field field) {
   private Object generateFixedType(Schema localSchema) {
     // TODO: Need to implement valid data generation for fixed type
     GenericFixed genericFixed = new GenericData.Fixed(localSchema);
-    switch (localSchema.getLogicalType().getName()) {
+    switch (localSchema.getType().getName()) {
```

Comment:
why this change?

Reply:
Same, my intellij doesn't work, reverted